changelog/fix_some_false_positives_by_replacing.md

```diff
@@ -0,0 +1 @@
+* [#866](https://github.com/rubocop/rubocop-rails/pull/866): Make `Rails/DurationArithmetic`, `Rails/IndexBy`, `Rails/IndexWIth`, and `Rails/RequireDependency` cops aware of const that starts with `::`. ([@r7kamura][])
```

Comment:
How about like this?

```suggestion
* [#866](https://github.com/rubocop/rubocop-rails/pull/866): Fix false-positives on `Rails/DurationArithmetic`, `Rails/IndexBy`, `Rails/IndexWIth`, and `Rails/RequireDependency` cops. ([@r7kamura][])
```

Reply:
Sure. It would be even better if there were additional explanations of what false positives were fixed.